autopush/integration/app/server.js

```diff
@@ -4,23 +4,17 @@ var path = require("path");
 
 var port = process.env.PORT || 3000;
 
-app.use("/test", express.static(path.join(__dirname, "test")));
+// Set the current directory to a static folder since we'll need to access 'converter.js' from our webpage.
+app.use("/app", express.static(__dirname));
```

Comment:
This was the main trick. We're using this odd looking static folder because our HTML file will need access to the /autopush/integration/app/converter.js file above. Yeah, we probably could have just created a route for the one file instead of opening the entire folder... :shrug: